packages/odyssey-react-mui/src/captureConsoleError.ts

```diff
@@ -0,0 +1,28 @@
+/*!
+ * Copyright (c) 2024-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+/** Useful for unit testing when you don't want to pollute the terminal with error messages. */
+export const captureConsoleError = ({
```

Comment:
is this only used by the tests? @KevinGhadyani-Okta

Reply:
Yep! I wonder if there's a good place to put test utils.

Reply:
It's gone now: c6b5f857c0017488d10c10d16a1c45285ba26f51